CefSharp.WinForms/WebBrowserExtensions.cs

```diff
@@ -57,5 +59,46 @@ public static bool DestroyWindow(this IWebBrowser chromiumWebBrowser)
 
             return DestroyWindow(handle);
         }
+
+        /// <summary>
+        /// Open developer tools using <paramref name="parentControl"/> as the parent control. If inspectElementAtX and/or inspectElementAtY are specified then
+        /// the element at the specified (x,y) location will be inspected.
+        /// </summary>
+        /// <param name="chromiumWebBrowser"><see cref="ChromiumWebBrowser"/> instance</param>
+        /// <param name="parentControl">Control used as the parent for DevTools (a custom control will be added to the <see cref="Control.Controls"/> collection)</param>
+        /// <param name="inspectElementAtX">x coordinate (used for inspectElement)</param>
+        /// <param name="inspectElementAtY">y coordinate (used for inspectElement)</param>
+        /// <returns>Returns the <see cref="Control"/> that hosts the DevTools instance if successful, otherwise returns null on error.</returns>
+        public static Control ShowDevToolsDocked(this IWebBrowser chromiumWebBrowser, Control parentControl, string controlName = nameof(ChromiumHostControl) + "DevTools", DockStyle dockStyle = DockStyle.Fill, int inspectElementAtX = 0, int inspectElementAtY = 0)
+        {
+            if(chromiumWebBrowser.IsDisposed || parentControl == null || parentControl.IsDisposed)
+            {
+                return null;
+            }
+
+            var host = chromiumWebBrowser.GetBrowserHost();
+            if(host == null)
+            {
+                return null;
+            }
+
+            var control = new ChromiumHostControl()
+            {
+                Name = controlName,
+                Dock = dockStyle
+            };
+
+            control.CreateControl();
+
+            parentControl.Controls.Add(control);
```

Comment:
Maybe instead of a `parentControl` parameter use an `Action<ChromiumHostControl / Control>` instead so the caller needs to add it themselves, which allows them to put it directly into some layout with the correct arguments.
Similar to here https://github.com/cefsharp/CefSharp/pull/3529/files#diff-1bb5944c3cd6f7978e9c18bc206e2e6fa23a98a69df010b612279631e7af1c0bR134

Reply:
Interesting, thanks 👍 

Should be able to provide an overload for additional flexibility, I think keeping the super simple version also makes sense.

Reply:
An overload was added in commit https://github.com/cefsharp/CefSharp/pull/3529/commits/4a331bc6531593a6949f6f434c1a2c8106846978#diff-e2ef0415e94478a46a76eeba9307a7adf47afca09963cb69c4cfb335d103414cR74